docs/references/ic-interface-spec.md

```diff
@@ -2083,13 +2082,10 @@ These system calls return costs in Cycles, represented by 128 bits, which will b
 
     See [`sign_with_ecdsa`](#ic-sign_with_ecdsa), [`sign_with_schnorr`](#ic-sign_with_schnorr) and [`vetkd_encrypted_key`](#ic-vetkd_encrypted_key) for more information.
 
-    These system calls trap if the string represented by `src` + `size` does not correspond to a valid key name, such as `dfx_test_key`, `test_key_1` or `key_1`, or if the provided curve enum is an unspecified variant. 
-
--   `ic0.replication_factor : (src : I, size : I) -> i32`; `I ∈ {i32, i64}`
-
-    Returns the replication factor (subnet size) of the subnet identified by the `Principal` at `src` + `size`. 
-
-    This system call traps if `src` + `size` do not represent a valid `Principal` or the given `Principal` is not a subnet.   
+    These system calls trap if `src` + `size` exceeds the size of the WebAssembly memory. Otherwise, they return an `i32` with the following meaning:
```

Comment:
```suggestion
    These system calls trap if `src` + `size`  or `dst` + 8 exceed the size of the WebAssembly memory. Otherwise, they return an `i32` with the following meaning:
```